site/en/docs/extensions/reference/userscript/index.md

```diff
@@ -0,0 +1,46 @@
+---
+api: userScript
+---
+
+## Using now
+
+This is an experimental feature that is not yet available in Chrome by default. You must take extra steps to enable it for experimentation.
+
+To enable it from a command line, use the following flag with the appropriate [command for your operating system](https://www.chromium.org/developers/how-tos/run-chromium-with-flags/).
+
+`--enable-features="ApiUserScripts"`
+
+## More information and feedback
+
+You can find more information about this [feature in the explainer](https://github.com/w3c/webextensions/blob/main/proposals/user-scripts-api.md). If you have feedback on the feature or find something that doesn't work, file a bug.
+
+## Manifest
+
+To use the `chrome.userScripts` API, you need to specify a `"manifest_version"` of `3` or higher. The manifest must include the `"userScripts"` permission and `"host_permissions"` for sites you want to run scripts on.
```

Comment:
I think you can use the activeTab as well, like in the [Scripting API](https://developer.chrome.com/docs/extensions/reference/scripting/#manifest). I'm confirming with Oliver.

Reply:
Do you have an answer?

Reply:
Let's just merge with host permissions. We can double-check the "activeTab" permission later and add it on a separate PR